### PR TITLE
Fix account groups spacing

### PR DESCRIPTION
**Pull request by serjonya-trili**
## Proposed changes

I split classes into several files and fixed the spacing between groups according to figma

## Types of changes

- [ ] Bugfix
- [ ] New feature
- [x] Refactor
- [ ] Breaking change
- [x] UI fix

## Screenshots

| Before | Now |
| ------ | --- |
| Screenshot 2024-03-19 at 14 46 56  |  Screenshot 2024-03-19 at 14 46 47   |

## Checklist

- [ ] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [x] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: src/views/home/getAccountGroupLabel.ts
Comment (asiia-trilitech) [inline src/views/home/getAccountGroupLabel.ts:1]: nit: is there a specific reason to move this method to a separate file? E. g. is it used more than in one place?

Happy for it to stay as is, just seems a bit excessive imo :)

File: src/views/home/AccountsList.test.tsx
Comment (asiia-trilitech) [inline src/views/home/AccountsList.test.tsx:1]: nit: please consider reorganising tests after refactoring (or creating a ticket for that 🙂).

Adding tests for separate (smaller) classes can ease writing test cases & provide better coverage.

**Review by asiia-trilitech — APPROVED** [review]
Looks good!

Thank you for cleaning up the codebase 👍

File: src/views/home/getAccountGroupLabel.ts
Comment (serjonya-trili) [reply src/views/home/getAccountGroupLabel.ts:1]: it is used in 2 places

**Pull request merged**